### ISSUE TITLE
Find a solution for longrunning tests that will work with future gradle versions

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-extras#168]
building with gradle 4.1 gives

```
> Configure project :org.eclipse.xtext.common.types.tests
Gradle now uses separate output directories for each JVM language, but this build assumes a single directory for all classes from a source set. This behaviour has been deprecated and is scheduled to be removed in Gradle 5.0
        at longrunning_tests_a0ntb8zioyd507ih16m3yv3ua$_run_closure5.doCall(/Users/dietrich/xtext-dev/git/xtext-extras/gradle/longrunning-tests.gradle:48)
The setTestClassesDir(File) method has been deprecated and is scheduled to be removed in Gradle 5.0. Please use the setTestClassesDirs(FileCollection) method instead.
        at longrunning_tests_a0ntb8zioyd507ih16m3yv3ua$_run_closure5.doCall(/Users/dietrich/xtext-dev/git/xtext-extras/gradle/longrunning-tests.gradle:48)
```
xtext-xtend might be affected as well

**Comment by cdietrich** [comment]
changing the code to `testClassesDirs = sourceSets.longrunning.output.classesDirs` should do the trick (requires gradle 4x though)

**Comment by oehme** [comment]
That's right.

**Comment by cdietrich** [comment]
@oehme at other places (e.g. xtext-web) i get

```
Gradle now uses separate output directories for each JVM language, but this build assumes a single directory for all classes from a source set. This behaviour has been deprecated and is scheduled to be removed in Gradle 5.0
```

how can i find out what exactly is offending?

**Comment by oehme** [comment]
Run with `--stacktrace`. We'll improve this message soon.

**Comment by cdietrich** [comment]
Fixed in 2.16
